Mesh_3/include/CGAL/Mesh_3/Image_plus_weights_to_labeled_function_wrapper.h

```diff
@@ -38,7 +40,7 @@ class Weighted_indicator_factory
     , weights(weights)
   {}
 
-  class Indicator : public std::unary_function<Image_word_type, double>
+  class Indicator : public std::function<double(Image_word_type)>
```

Comment:
No!
  - `std::unary_function` was an empty class providing only a few nested types.
  - `std::function` is alike the old `boost::function`: a type-erasure utility to wrap a function-like object.

Reply:
should we simply remove the inheritance then? (I tried, it works)

Reply:
Yes, I think so.